src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -312,36 +313,66 @@ export class BrowsingContextProcessor {
   async process_script_evaluate(
     params: Script.EvaluateParameters
   ): Promise<Script.EvaluateResult> {
-    if ('realm' in params.target) {
-      throw new Error('Realm target is not implemented yet.');
-    }
-    const target: Script.ContextTarget = params.target;
+    const scriptTarget = BrowsingContextProcessor.#getScriptTarget(
+      params.target
+    );
 
-    const context = BrowsingContextProcessor.#getKnownContext(target.context);
-    return await context.scriptEvaluate(
+    return await scriptTarget.context.scriptEvaluate(
       params.expression,
-      target.sandbox ?? null,
+      scriptTarget.target,
       params.awaitPromise,
       params.resultOwnership ?? 'none'
     );
   }
 
+  static #getScriptTarget(target: Script.Target): {
+    context: BrowsingContextImpl;
+    target: ScriptTarget;
+  } {
+    if ('realm' in target) {
+      const { executionContextId, browsingContextId } = Realm.getRealm(
+        target.realm
+      );
+      return {
+        context: BrowsingContextProcessor.#getKnownContext(browsingContextId),
+        target: { executionContext: executionContextId },
+      };
+    } else {
+      return {
+        context: BrowsingContextProcessor.#getKnownContext(target.context),
+        target: { sandbox: target.sandbox ?? null },
+      };
+    }
+  }
+
+  process_script_getRealms(
+    params: Script.GetRealmsParameters
+  ): Script.GetRealmsResult {
+    if (params.context !== undefined) {
+      // Make sure the context is known.
+      BrowsingContextProcessor.#getKnownContext(params.context);
+    }
+    const realms = Realm.findRealms({
+      browsingContextId: params.context,
+      type: params.type,
+    }).map((realm) => realm.toBiDi());
+    return { result: { realms } };
+  }
+
   async process_script_callFunction(
     params: Script.CallFunctionParameters
   ): Promise<Script.CallFunctionResult> {
-    if ('realm' in params.target) {
-      throw new Error('Realm target is not implemented yet.');
-    }
-    const target: Script.ContextTarget = params.target;
+    const scriptTarget = BrowsingContextProcessor.#getScriptTarget(
+      params.target
```

Comment:
```suggestion
      params.target,
```

Reply:
Prettier doesn't think so )

Reply:
This is a style discussion. If we want trailing commas for function params across multiple lines we could set `trailingComma: 'all'` in the Prettier config. But I agree that we should discuss that separately from the code changes in this PR 👍